test/src/test/java/com/distkv/pine/CacheTest.java

```diff
@@ -0,0 +1,38 @@
+package com.distkv.pine;
+
+import com.distkv.client.DistkvClient;
+import com.distkv.common.exception.KeyNotFoundException;
+import com.distkv.common.utils.RuntimeUtil;
+import com.distkv.pine.api.Pine;
+import com.distkv.pine.components.cache.PineCache;
+import com.distkv.supplier.BaseTestSupplier;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class CacheTest extends BaseTestSupplier {
+
+  private DistkvClient distkvClient;
+
+  @Test
+  public void testCache() throws InterruptedException {
+    Pine.init(getListeningAddress());
+
+    PineCache cache = Pine.newCache((long) 1000);
+    cache.newItem("zhangsan");
+    cache.newItem("lisi");
+    //test  key not expired
+    Assert.assertFalse(cache.isExpired("zhangsan"));
+
+    //test key expired
+    boolean flg = RuntimeUtil.waitForCondition(() -> {
+      try {
+        return cache.isExpired("wuzi");
```

Comment:
Why wuzi?  Why not lisi or zhangsan?

Reply:
here  purpose to test  the key which never appear in cache。so “wuzi” is never appear .
there will trigger keyNotfoundException

Reply:
But  this UT case not test key expired.